common/src/test/groovy/com/okta/cli/common/model/OidcPropertiesTest.groovy

```diff
@@ -63,6 +67,95 @@ class OidcPropertiesTest {
         oidcProperties.setRedirectUris(List.of("http://localhost:8080/login/oauth2/code/oidc"))
         def clientProperties3 = oidcProperties.getProperties()
         assertThat clientProperties3.get("quarkus.oidc.authentication.redirect-path"), is("/login/oauth2/code/oidc")
+    }
+
+    @Test
+    void jhipsterQuarkusServicePropertiesDetectionTest() {
+
+        String packageJson ="""
+            "devDependencies": {
+                "generator-jhipster": "6.10.5",
+                "generator-jhipster-quarkus": "1.0.0",
+            }
+        """.stripLeading()
+
+        Map<String, String> expected = [
+            "quarkus.oidc.auth-server-url": "https://issuer.example.com",
+            "quarkus.oidc.client-id": "test-client-id",
+            "quarkus.oidc.credentials.secret": "test-client-secret",
+            "quarkus.oidc.authentication.redirect-path": "/login/oauth2/code/oidc",
```

Comment:
I'm not sure we need to set this. The [default](https://github.com/jhipster/generator-jhipster-quarkus/blob/main/generators/server/templates/quarkus/src/main/resources/application.properties.ejs#L90) is the same value.

```
quarkus.oidc.authentication.redirect-path=/login/oauth2/code/oidc
```